File: python3/rpc/wtf.py
#!/usr/bin/env python3
# coding: utf-8
#


import statistics


def main():
    arr = []
    fn = 'data.txt'
    try:
        with open(fn, 'rt') as ifile:
            for ln in ifile:
                val = float(ln.strip())
                arr.append(val)
    except IOError:
        print('IOError')
        return

    print('there are {} elements'.format(len(arr)))
    mean = statistics.mean(arr)
    median = statistics.median(arr)
    stdev = statistics.stdev(arr)
    print('mean: {:.3f}\nmedian: {:.3f}\nstdev: {:.3f}\n'.format(mean, median, stdev))


if __name__ == '__main__':
    main()

File: Topics/boshiamy/wtf.pl
#!/usr/bin/perl
#
use strict;

while (<DATA>) {
    s/[\r\n]//;
    if ( m/^\w+:\s+(\d+)/ ) {
        printf("%x\n", $1);
    }
}

__DATA__
missing: 131205
missing: 139304
missing: 139305
missing: 173783
missing: 173784
missing: 173785
missing: 173786
missing: 173787
missing: 173788
missing: 173789
missing: 173790
missing: 173791
